lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -742,7 +738,7 @@ public function getMigrationsToExecute($direction, $to)
         }
 
         if ($direction === Version::DIRECTION_DOWN) {
-            if (count($this->migrations)) {
+            if ($this->migrations) {
```

Comment:
sure it works the same. We are talking about explicitness to make the code more readable